test/unit/math/src/math.cpp

```diff
@@ -125,7 +125,7 @@ struct TestTemplate
         {
             INFO("Idx i: " << i)
             TData std_result = functor(args(i));
-            REQUIRE(results(i) == Approx(std_result));
+            REQUIRE(results(i) == Approx(std_result).margin(std::numeric_limits<TData>::epsilon()));
```

Comment:
I was first confused by this, since `Approx` already sets an epsilon of `std::numeric_limits<float>::epsilon()*100`. But then I saw you set a margin, which by default is 0, instead of an epsilon.